src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -154,6 +186,24 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val jobRunsSnapshotTarget: PipelineTable = PipelineTable(
+      name = "job_runs_snapshot_bronze",
+      _keys = Array("job_id", "run_id", "Overwatch_RunID"),
+      config,
+      incrementalColumns = Array("Pipeline_SnapTS"),
+      statsColumns = "created_time, creator_user_name, job_id, run_id, Pipeline_SnapTS, Overwatch_RunID".split(", "),
+      partitionBy = Seq("organization_id"),
+      masterSchema = Some(Schema.jobSnapMinimumSchema)
```

Comment:
this is the jobSnapMinSchema -- you should remove this for now as we don't have downstream dependencies so we don't have a master schema for this yet.

Reply:
removed `Some(Schema.jobSnapMinimumSchema)`